tests/testthat/_snaps/rename-files.md

```diff
@@ -5,46 +5,40 @@
     Message
       x Did not rename files!
       ! Found references to "data/my-streets.csv" in project
-      i Change file path to "data/my-roads.csv" or see `Find in Files` Replace All if
-        confident. Copied new name to clipboard
-      i Also change object names to snake_case that follow the new file name.
-      i Found 2 references to "my-streets" in 'R/my-analysis.R:1:0' and
-        'R/my-analysis.R:2:11'.
-      Rerun the code to make it work or use `force = TRUE`
+      i Change file path to "data/my-roads.csv" in files ahead of renaming file or see `Find in Files` Replace All if confident. Copied new name to clipboard
+      Also change object names to snake_case that follow the new file name.
+      i Found 2 references to "my-streets/my_streets" in 'R/my-analysis.R:1:0' and 'R/my-analysis.R:2:11'.
     Output
       [1] "data/my-roads.csv"
 
 # rename_files2(): is easier to test messages with no action
 
     Code
-      rename_files2("data/my-streets.csv", "data/my-roads.csv", force = TRUE, action = "test")
+      rename_files2("data/my-streets.csv", "data/my-roads.csv", overwrite = TRUE,
+        action = "test")
     Message
-      Here are the conflicts. Review changes carefully
-      renaming file anyway
-      i Found 2 references to "my-streets" in 'R/my-analysis.R:1:0' and
-        'R/my-analysis.R:2:11'.
-      Testing mode, did not rename file
-      i Call `reuseme::check_referenced_files()` to see if there are dead links in dir.
+      x Did not rename files!
+      ! Found references to "data/my-streets.csv" in project
+      i Change file path to "data/my-roads.csv" in files ahead of renaming file or see `Find in Files` Replace All if confident. Copied new name to clipboard
+      Also change object names to snake_case that follow the new file name.
+      i Found 2 references to "my-streets/my_streets" in 'R/my-analysis.R:1:0' and 'R/my-analysis.R:2:11'.
+    Output
+      [1] "data/my-roads.csv"
 
 # rename_files2(): renames files if forced to do so
 
     Code
-      rename_files2("data/my-streets.csv", "data/my-roads.csv", force = TRUE)
+      rename_files2("data/my-streets.csv", "data/my-roads.csv", warn_conflicts = "none",
+        overwrite = TRUE)
     Message
-      Here are the conflicts. Review changes carefully
```

Comment:
these messages should not have been removed

Reply:
It is fine. To use `warn_conflict = "none"` suggests you may have done a pass before.